x-pack/plugins/security/server/lib/authorization/privileges.ts

```diff
@@ -57,8 +57,8 @@ export function privilegesFactory(
             actions.app.all,
             ...actions.savedObject.readOperations(validSavedObjectTypes),
             ...featuresPrivilegesBuilder.getUIReadActions(features),
-            ...featuresPrivilegesBuilder.getManagementReadActions(features),
-            ...featuresPrivilegesBuilder.getCatalogueReadActions(features),
+            actions.ui.allCatalogueEntries,
```

Comment:
I'm trying to recall if we had a reason why we weren't doing this before. Is it because we didn't want the "advanced settings" and "index patterns" screens to show up in Management for "read-only" users?

Reply:
Possibly -- I was torn on this too. These are read-only superusers, if that makes any sense at all, so I opted to include them...do you think there's benefit to allowing these users to read settings and index patterns?

Reply:
I think we get benefit from this, but I believe we were trying to prevent having to make read-only views of these screens or having them throw ugly errors when they go to save.

Reply:
Agreed, and I don't want to add that to our scope. As-is, they would likely get ugly errors when trying to save. 
So I guess we have to decide which is the lesser of two evils: Would we rather hide these for read-only users, even though they _should_ have read access, or do we show them, knowing that they'll have a bad experience if they try to create/update?

Reply:
I think so, yeah...

Reply:
I'm leaning towards not showing them for read-only users, do you have a preference?

Reply:
that works for me